ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -18,4 +19,21 @@ import_config:
     # A map from versions to dump and their output names.
     versions_to_import:
       123: testnet_v1.json
+# Config to generate the transactions via localnode.
+script_transaction_generator_config:
+  runs:
+    - transactions:
+      - output_name: transfer_from_a_to_b
+        script_path: path/to/script1
+        fund_address: address_a
+      - output_name: transfer_from_b_to_c
+        script_path: path/to/script2
+    - transactions:
+      # Note: we've generated transactions for script1, we don't need the name anymore.
```

Comment:
in this case, do we just not generate another transaction file even though the transaction still gets submitted through the network?

Reply:
yes, this is just for dependent transactions. for individual transactions, can be submitted in one run.